internal/evidence/pool_test.go

```diff
@@ -141,8 +148,14 @@ func TestReportConflictingVotes(t *testing.T) {
 	var height int64 = 10
 
 	pool, pv := defaultTestPool(t, height)
-	val := types.NewValidator(pv.PrivKey.PubKey(), 10)
-	ev := types.NewMockDuplicateVoteEvidenceWithValidator(height+1, defaultEvidenceTime, pv, evidenceChainID)
+
+	quorumHash, err := pv.GetFirstQuorumHash(context.Background())
```

Comment:
this call does into `func TestReportConflictingVotes(t *testing.T)` test case. I think it would be better to get quorum-hash from a state `pool.State().Validators.QuorumHash`, wdyt?